README.md

```diff
@@ -25,136 +25,33 @@
 using modern software design and is parallelized with MPI for distributed memory hardware architectures.
 
 **Disclaimer**: 4C is developed for research purposes in the field of numerical method development.
-It is not intended for any use beyond this purpose, and generally should not be used for any form of
-safety-relevant or safety-critical calculations,
+It is not intended for any use beyond this purpose and generally should not be used for any form of
+safety-relevant or safety-critical calculations
 or for an application in association with physical products in particular.
 
-## Required development tools
+## Getting started
 
-- **CMake:** 4C supports CMake configuration through CMake presets.
-- **Ninja:** We strongly encourage to work with `ninja` (instead of `make`) for faster compile times.
-
-## External dependencies
-
-4C heavily relies on the [Trilinos project](https://trilinos.github.io).
-
-Some further third-party libraries (TPLs) are mandatory, e.g.
-
-- CMake (minimum version: 3.30)
-- ParMETIS (recommended version: 4.0.3)
-- SuiteSparse (recommended version: 5.4.0)
-- SuperLUDist (recommended version: 7.2)
-- Qhull (recommended version: 2012.1)
-- CLN (recommended version: 1.3.4)
-
-and some are optional, e.g.
-
-- FFTW
-- [ArborX](https://github.com/arborx/ArborX)
-- [MIRCO](https://github.com/imcs-compsim/MIRCO/)
-
-Maybe, a pre-compiled version of Trilinos and set of TPLs is available at your institute.
-Look into the CMake presets in `presets/` or ask your colleagues for further information.
-
-Some helper scripts to install these TPLs can be found in `dependencies/`.
-
-Additional information can be found in
-the [user documentation](https://4c-multiphysics.github.io/4C/readthedocs/4Csetup.html#external-dependencies).
-
-## Getting up and running
-
-### Create python virtual environment (required for code development in 4C)
-
-For testing and active development, you need to create a python virtual environment once.
-In the source directory,
-execute:
-
-```
-./utilities/set_up_dev_env.sh
-```
-
-### Configure and Build
-
-4C enforces an out-of-source build, i.e., the build directory may not be the top-level source directory.
-Create a build directory `<path/to/build/dir>` of your choosing.
-
-Navigate into the build directory and run
-
-```bash
-cmake --preset=<name-of-preset> <path/to/source/dir>
-```
-
-A preset name needs to be passed to cmake via the command line argument `--preset`.
-Use `cmake <path/to/source/dir> --list-presets` to get a list of all available presets.
-Define your own presets in a `CMakeUserPresets.json` file.
-
-More information about CMake presets can be found in the [CMake documentation](https://cmake.org/cmake/help/latest/manual/cmake-presets.7.html).
-
-#### Build
-
-To build the code on `<numProcs>` processes, run
-
-```bash
-ninja -j <numProcs> full
+To quickly run 4C, you can use our docker image, where 4C comes pre-compiled.
 ```
-
-#### Run the Tests
```

Comment:
I'm not too happy about docker. It makes it very easy to use 4C. However, it is pretty cumbersome to get the simulation results out of the docker container.